src/utils/export-map.ts

```diff
@@ -67,6 +67,21 @@ const declTypes = new Set([
   'TSModuleDeclaration',
 ])
 
+// https://github.com/syavorsky/comment-parser/issues/172
+const fixup = new Set(['deprecated', 'module'])
+
+const parseComment = (comment: string): Block => {
+  const [doc] = parseComment_(comment)
+  return {
+    ...doc,
+    tags: doc.tags.map(t =>
+      fixup.has(t.tag) && t.name
```

Comment:
```suggestion
      t.name && fixup.has(t.tag)
```

`t.name` falsy or not can be checked faster than `set.has`. We could bail out early without doing expensive `set.has`.